# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../qemu/qemu-0.8.3-enforce-16byte-boundaries.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

2006-02-12  Gwenole Beauchesne  <gbeauchesne@mandriva.com>

	* Enforce 16-byte boundaries.

--- qemu-0.8.3/Makefile.target.enforce-16byte-boundaries	2007-01-24 22:40:21.000000000 +0100
+++ qemu-0.8.3/Makefile.target	2007-01-30 18:23:10.000000000 +0100
@@ -74,7 +74,7 @@ OP_CFLAGS = -Wall -O2 -g -fno-strict-ali
 
 ifeq ($(ARCH),i386)
 HELPER_CFLAGS+=-fomit-frame-pointer
-OP_CFLAGS+=-mpreferred-stack-boundary=2 -fomit-frame-pointer
+OP_CFLAGS+=-fomit-frame-pointer
 ifeq ($(HAVE_GCC3_OPTIONS),yes)
 OP_CFLAGS+= -falign-functions=0 -fno-gcse
 else
